R/read_sourmash_outputs.R

```diff
@@ -1,3 +1,20 @@
+#' Check if local file path or URL are valid
+#'
+#' @param file File path or URL
+#'
+#' @return NULL
+#'
+#' @examples
+check_file_and_url_one <- function(file){
```

Comment:
probably not worth it now, but may be worth separating this into a `utils/` directory or smth

Reply:
ok I'll keep this in mind for the future and move toward that!